tests/foreman/cli/test_docker.py

```diff
@@ -998,18 +1157,9 @@ class DockerComputeResourceTestCase(CLITestCase):
     def setUpClass(cls):
         """Create an organization and product which can be re-used in tests."""
         super(DockerComputeResourceTestCase, cls).setUpClass()
-        cls.org_id = entities.Organization().create_json()['id']
+        cls.org = make_org()
 
-    @stubbed()
-    @run_only_on('sat')
-    @data(
-        gen_string('alpha'),
-        gen_string('alphanumeric'),
-        gen_string('numeric'),
-        gen_string('latin1'),
-        gen_string('utf8'),
-        gen_string('html'),
-    )
+    @data(*valid_data_list())
```

Comment:
`test_create_internal_docker_compute_resource` and `test_create_external_docker_compute_resource` are doing basically the same for different urls. does anyone have ideas how to correctly merge 2 lists of different length in data decorator? i was thinking about smth like 
```python
@data(zip(valid_data_list(), cycle([INTERNAL_DOCKER_URL, EXTERNAL_DOCKER_URL]))
```
but it doesn't look readable :) and needs `itertools` to be imported
of course, i could do smth like
```python
          {'name': gen_string('alphanumeric', randint(1, 255)),
           'url': INTERNAL_DOCKER_URL},
          {'name': gen_string('alpha', randint(1, 255)),
           'url': INTERNAL_DOCKER_URL},
          {'name': gen_string('cjk', randint(1, 85)),
           'url': INTERNAL_DOCKER_URL},
          {'name': gen_string('latin1', randint(1, 255)),
           'url': INTERNAL_DOCKER_URL},
          {'name': gen_string('numeric', randint(1, 255)),
           'url': INTERNAL_DOCKER_URL},
          {'name': gen_string('utf8', randint(1, 85)),
           'url': INTERNAL_DOCKER_URL},
          {'name': gen_string('html', randint(1, 85)),
           'url': INTERNAL_DOCKER_URL},
          {'name': gen_string('alphanumeric', randint(1, 255)),
           'url': EXTERNAL_DOCKER_URL},
          {'name': gen_string('alpha', randint(1, 255)),
           'url': EXTERNAL_DOCKER_URL},
          {'name': gen_string('cjk', randint(1, 85)),
           'url': EXTERNAL_DOCKER_URL},
          {'name': gen_string('latin1', randint(1, 255)),
           'url': EXTERNAL_DOCKER_URL},
          {'name': gen_string('numeric', randint(1, 255)),
           'url': EXTERNAL_DOCKER_URL},
          {'name': gen_string('utf8', randint(1, 85)),
           'url': EXTERNAL_DOCKER_URL},
          {'name': gen_string('html', randint(1, 85)),
           'url': EXTERNAL_DOCKER_URL},)
```
but this doesn't look like the best solution either. is there a better way? or maybe i should leave 2 tests as they are?